### DISCUSSION TITLE
Can we enable bertopic with Synapse ML

### DISCUSSION THREAD

**Discussion by joshedOpenAi** [MaartenGr/BERTopic#1754] [Q&A]
Is there a way to use bertopic with synapse ml https://microsoft.github.io/SynapseML/docs/next/Explore%20Algorithms/OpenAI/

**Comment by MaartenGr** [comment]
It is currently not implemented but it should be rather straightforward to implement it yourself using some of the examples here: https://github.com/MaartenGr/BERTopic/tree/master/bertopic/representation
